code/parse/tables/ai_class_info.cpp

```diff
@@ -0,0 +1,108 @@
+#include "ai_class_info.h"
```

Comment:
I would prefer to keep the table definition close to the code that also parses the table so this would be better to have somewhere in the AI code.

Reply:
It should be moved now.